### PR TITLE
convert to markdown for orphaned folder 

### PR DESCRIPTION
**Pull request by yin1999**
### Description

convert to markdown for orphaned folder.

### Related issues and pull requests

- https://github.com/orgs/mdn/discussions/227
- mdn/yari#7574

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
Also manually checked those rendered documents. Seems ok for me

File: files/ru/orphaned/glossary/bootstrap/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+  - Bootstrap
+  - CSS
+  - Введение
+  - Глоссарий
+  - фреймворк
+translation_of: Glossary/Bootstrap
+original_slug: Glossary/Bootstrap
+---
+Bootstrap - это бесплатный фреймворк {{Glossary("HTML")}} , {{Glossary("CSS")}} и {{Glossary("JavaScript")}} с открытым исходным кодом для быстрого создания адаптивных веб-сайтов.
```
Comment (sashasushko) [inline files/ru/orphaned/glossary/bootstrap/index.md:13]:
```suggestion
---

Bootstrap - это бесплатный фреймворк {{Glossary("HTML")}} , {{Glossary("CSS")}} и {{Glossary("JavaScript")}} с открытым исходным кодом для быстрого создания адаптивных веб-сайтов.
```

File: files/ru/orphaned/glossary/jquery/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: jQuery
+slug: orphaned/Glossary/jQuery
+translation_of: Glossary/jQuery
+original_slug: Glossary/jQuery
+---
+**jQuery** это {{Glossary("JavaScript")}} {{Glossary("Библиотека")}} сфокусированная на управлении {{Glossary("DOM")}} , вызовах {{Glossary("AJAX")}} , и {{Glossary("событиях")}} обработки.
```
Comment (sashasushko) [inline files/ru/orphaned/glossary/jquery/index.md:7]:
```suggestion
---

**jQuery** это {{Glossary("JavaScript")}} {{Glossary("Библиотека")}} сфокусированная на управлении {{Glossary("DOM")}} , вызовах {{Glossary("AJAX")}} , и {{Glossary("событиях")}} обработки.
```

File: files/ru/orphaned/web/api/document/documenturiobject/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: Document.documentURIObject
+slug: orphaned/Web/API/Document/documentURIObject
+tags:
+  - свойства DOM
+translation_of: Web/API/Document/documentURIObject
+original_slug: Web/API/Document/documentURIObject
+---
+{{ ApiRef("DOM") }}
```
Comment (sashasushko) [inline files/ru/orphaned/web/api/document/documenturiobject/index.md:9]:
```suggestion
---

{{ ApiRef("DOM") }}
```

File: files/ru/orphaned/web/api/navigatorid/platform/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  - API
+  - Navigator
+  - NavigatorID
+  - Property
+  - Refe
+  - platform
+translation_of: Web/API/NavigatorID/platform
+original_slug: Web/API/NavigatorID/platform
+---
+{{ APIRef("HTML DOM") }}
```
Comment (sashasushko) [inline files/ru/orphaned/web/api/navigatorid/platform/index.md:14]:
```suggestion
---

{{ APIRef("HTML DOM") }}
```

File: files/ru/orphaned/web/api/navigatorlanguage/languages/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+title: NavigatorLanguage.languages
+slug: orphaned/Web/API/NavigatorLanguage/languages
+translation_of: Web/API/NavigatorLanguage/languages
+original_slug: Web/API/NavigatorLanguage/languages
+---
+{{APIRef("HTML DOM")}}{{SeeCompatTable}}
```
Comment (sashasushko) [inline files/ru/orphaned/web/api/navigatorlanguage/languages/index.md:7]:
```suggestion
---

{{APIRef("HTML DOM")}}{{SeeCompatTable}}
```

File: files/ru/orphaned/web/guide/unicode_bidrectional_text_algorithm/index.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+tags:
+  - Алгоритм
+  - Локализация
+  - Руководство
+  - Юникод
+  - направление
+translation_of: Web/Guide/Unicode_Bidrectional_Text_Algorithm
+original_slug: Web/Guide/Unicode_Bidrectional_Text_Algorithm
+---
+**Алгоритм обработки текста двусторонней направленности** (Unicode® Bidirectional Algorithm или **BiDi** Algorithm) — раздел стандарта текста Unicode, описывающий в каком порядке раскладываются символы во время обработки страницы {{Glossary("агентом пользователя")}}.
```
Comment (sashasushko) [inline files/ru/orphaned/web/guide/unicode_bidrectional_text_algorithm/index.md:13]:
```suggestion
---

**Алгоритм обработки текста двусторонней направленности** (Unicode® Bidirectional Algorithm или **BiDi** Algorithm) — раздел стандарта текста Unicode, описывающий в каком порядке раскладываются символы во время обработки страницы {{Glossary("агентом пользователя")}}.
```

File: files/ru/orphaned/web/javascript/reference/errors/var_hides_argument/index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+slug: orphaned/Web/JavaScript/Reference/Errors/Var_hides_argument
+tags:
+  - Errors
+  - JavaScript
+  - Strict Mode
+  - TypeError
+translation_of: Web/JavaScript/Reference/Errors/Var_hides_argument
+original_slug: Web/JavaScript/Reference/Errors/Var_hides_argument
+---
+{{jsSidebar("Errors")}}
```
Comment (sashasushko) [inline files/ru/orphaned/web/javascript/reference/errors/var_hides_argument/index.md:12]:
```suggestion
---

{{jsSidebar("Errors")}}
```

**Review by sashasushko — APPROVED** [review]
LGTM 🔥

**Pull request merged**